css/css-sizing/aspect-ratio/flex-aspect-ratio-027.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<title>CSS aspect-ratio: auto size img with aspect-ratio inside column flexbox</title>
+<link rel="help" href="https://www.w3.org/TR/css-sizing-4/#propdef-aspect-ratio">
+<link rel="help" href="https://www.w3.org/TR/css-flexbox-1/#layout-algorithm">
+<link rel="match" href="../../reference/ref-filled-green-100px-square.xht" />
+
+<p>Test passes if there is a filled green square and <strong>no red</strong>.</p>
+<style>
+    div {
+        width: 100px;
+        display: flex;
+        flex-direction: column;
+    }
+    img {
+        max-height: 100%;
+        max-width: 100%;
+        width: auto;
+        height: auto;
```

Comment:
Order is usual width, then height. Can any of these be removed, for instance the auto?

Reply:
Done. Yeah, we should remove this.